FEATURES.md

```diff
@@ -0,0 +1,113 @@
+# Features
+
+- [Document Symbol](#document-symbol)
+    - [Active Record DSLs and Associations](#active-record-dsls-and-associations)
+    - [Active Support Test Cases](#active-support-test-cases)
+- [Go to Controller Action Route](#go-to-controller-action-route)
+- [Go to Controller Action View](#go-to-controller-action-view)
+- [Go to Definition](#go-to-definition)
+    - [Go to Active Record DSL Definitions](#go-to-active-record-dsl-definitions)
+    - [Go to Active Record Associations](#go-to-active-record-associations)
+    - [Go to Route Helper Definitions](#go-to-route-helper-definitions)
+- [Ruby File Operations](#ruby-file-operations)
+    - [Commands](#commands)
+        - [Rails Generate](#rails-generate)
+        - [Rails Destroy](#rails-destroy)
+    - [Icon](#icon)
+- [Run and Debug](#run-and-debug)
+    - [Run Tests With Test Explorer](#run-tests-with-test-explorer)
+    - [Run Tests In The Terminal](#run-tests-in-the-terminal)
+    - [Debug Tests With VS Code](#debug-tests-with-vs-code)
+
+## Document Symbol
+
+Document Symbol is a way to represent the structure of a document. They are used to provide a quick overview of the
+document and to allow for quick navigation.
+
+Ruby LSP already provides document symbols for Ruby files, such as classes, modules, methods, etc. But the Rails addon
+provides additional document symbols for Rails specific features.
+
+In VS Code, you can open the document symbols view by pressing `Ctrl + Shift + O`.
+
+### Active Record DSLs and Associations
+
+Navigates between Active Record DSLs and Associations using the `Document Symbol` feature.
+
+![Document Symbol for Active Record DSLs and Associations](./ruby-lsp-rails-document-symbol-ar-model.gif)
```

Comment:
Can we explicitly say callbacks, validations and associations instead of just DSLs? I'm afraid not everyone reading the documentation will have enough context to know what those are.